back_end/index.js

```diff
@@ -419,6 +419,163 @@ app.get('/fornecedor/:id', (req, res) => {
   console.log('get /fornecedor/' + fornecedorId + ' executado. Fornecedor retornado com sucesso!');
 });
   
+
+//alteração Erik e Marcos//
+
+app.get('/usuario', (req, res) => {
+  db.query('SELECT * FROM usuario', (err, results) => {
+    if (err) {
+      res.status(500).send('Erro retornando usuario: ' + err);
+      return;
+    }
+    res.json(results);
+  });
+  console.log('get /usuario executado. Usuarios retornados com sucesso!');
+});
+   
+/* POST */
+app.post('/usuario/create', (req, res) => {
```

Comment:
apenas POST /usuario ja supre a convenção, POST é usado sempre q for criar um recurso novo.

Reply:
Legal, é bom saber, mas eu manteria assim mesmo pra manter o padrao que estamos usando